### PR TITLE
Portfolio - Rosanna Dahlberg

### PR DESCRIPTION
**Pull request by Rosanna86**
Portfolio Project Week 8

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+                href="https://stackoverflow.com/users/16650705/rosanna-dahlberg">
+                <i class="fab fa-stack-overflow circle-icon"></i>
+            </a>
+        </div>
+        <div class="headings-info">
+            <h1 class="heading-1">PORTFOLIO: <span style="font-weight:bold">ROSANNA DAHLBERG</span></h1>
+            <h1 class="heading-2">frontend</h1>
+            <h1 class="heading-2 tight">developer</h1>
+            <h1 class="heading-3">+ graphic designer</h1>
+        </div>
```
Comment (DALA746) [inline code/index.html:44]: Really nice solution to deal with all of these headings by adding different classes to them 👍

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+                <span>HTML5</span>
+                <span>CSS3</span>
+                <span>JavaScript ES6</span>
+                <a class="tag-github" href="https://github.com/Rosanna86/project-chatbot">GitHub</a>
+            </div>
+        </div>
+    </section>
+
+    <!-- Thoughts -->
+    <section class="section section-blue centered">
```
Comment (DALA746) [inline code/index.html:361]: Really interesting approach to style with multiple classes :)

File: code/index.html [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+        <div class="heading-h2">
+            <h2 class="heading headingwhite">THOUGHTS</h2>
+        </div>
+        <p> Stay tuned... Soon something brilliant will appear!
+        </p>
+    </section>
+
+    <!-- Skills -->
+    <section class="section pad-tablet">
```
Comment (DALA746) [inline code/index.html:377]: Your skills text is a bit to the left.  If you want your text to the center of the section you can apply text-align: center to the section. :)

**Review by DALA746 — COMMENTED** [review]
Grid is really new for me and it was really interesting to see your code and how you can manipulate with grid technology. Your code follows the design we've been given and your page is nicely responsive. I really like the idea of using multiple classes that you used in your code. Otherwise, great job on this project and I'm looking forward to seeing more of your code in the future 💯

File: code/index.html (hunk 1 above)
Comment (Rosanna86) [reply code/index.html:44]: Thank you :)

File: code/index.html (hunk 3 above)
Comment (Rosanna86) [reply code/index.html:377]: Thank you! Will update that!

File: code/index.html (hunk 2 above)
Comment (Rosanna86) [reply code/index.html:361]: Yeah, that helped a lot!

File: code/index.html [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+                <p>Responsive web site using HTML5, CSS3 and JavaScript ES6 for coffee orders.
+                    <span class="arrow-other-prj"></span>
+                </p>
+            </a>
+            <div class="tag-wrapper">
+                <span>HTML5</span>
+                <span>CSS3</span>
+                <span>JavaScript ES6</span>
+                <a class="tag-github" href="https://github.com/Rosanna86/project-chatbot">GitHub</a>
+            </div>
```
Comment (AschwinSiaila) [inline code/index.html]: Looks good, the different color and link to github

File: code/index.html [hunk 5]
```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rosanna Dahlbergs portfolio</title>
+    <link rel="icon" href="./assets/heart-favicon.png" type="png">
```
Comment (AschwinSiaila) [inline code/index.html:9]: This was an error on the validator. I do not know what is wrong with it, but it should not bother the user I think. I saw an article where they wrote about this. The link is: https://css-tricks.com/weekly-platform-news-favicon-guidelines-accessibility-testing-web-almanac/

**Review by AschwinSiaila — COMMENTED** [review]
The portfolio looks very good and I am staying tuned for the brilliant things.

**Pull request closed without merging**